Enable new WebGL conformance tests, conformance/{context,extensions}

kkinnunen-apple opened the pull request:
#### d07a15d103fd10637522e84e5998e70d6a097476
<pre>
Enable new WebGL conformance tests, conformance/{context,extensions}
<a href="https://bugs.webkit.org/show_bug.cgi?id=254208">https://bugs.webkit.org/show_bug.cgi?id=254208</a>
rdar://106989736

Reviewed by Dean Jackson.

Enable subset of webgl/1.0.x, webgl/2.0.y and disable old tests.

* LayoutTests/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/262389@main">https://commits.webkit.org/262389@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/aafa95231d9e100e1081a1246a0b9bfeb2a1415a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/1234 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/1268 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/1307 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/2011 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/1103 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/1320 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/1336 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/1264 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/1243 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/1178 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/1144 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/1872 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/1180 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/1134 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/1117 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/1128 "Built successfully") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/1165 "6 failures") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/2228 "run-api-tests-without-change (failure)") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/1177 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/1082 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/1113 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/1144 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/350 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/1203 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

grorg reviewed (approved):
r=me once all the bots are happy :)

kkinnunen-apple committed:
Enable new WebGL conformance tests, conformance/{context,extensions}
https://bugs.webkit.org/show_bug.cgi?id=254208
rdar://106989736

Reviewed by Dean Jackson.

Enable subset of webgl/1.0.x, webgl/2.0.y and disable old tests.

* LayoutTests/TestExpectations:

Canonical link: https://commits.webkit.org/262389@main
